wallet/sig.go

```diff
@@ -24,8 +24,22 @@ import (
 	"perun.network/go-perun/wire/perunio"
 )
 
-// Sig is a single signature.
-type Sig = []byte
+// Sig represents a single signature generated over a given data using an
+// account. It is usually a byte array and the length depends upon the specific
+// signature scheme being used, which in turn depends upon the wallet backend.
+//
+// It is sent over wire as a binary data.
```

Comment:
This part is not useful to mention here as it explains the functionality of a different package.
```suggestion
//
// It is sent over wire as a binary data.
```